dataclients/kubernetes/definitions/routegroupvalidator.go

```diff
@@ -56,7 +63,10 @@ func (rgv *RouteGroupValidator) filtersValidation(item *RouteGroupItem) error {
 	var errs []error
 	for _, r := range item.Spec.Routes {
 		for _, f := range r.Filters {
-			_, err := eskip.ParseFilters(f)
+			filters, err := eskip.ParseFilters(f)
+			if len(filters) > 1 {
+				errs = append(errs, errMultipleFilters)
+			}
```

Comment:
I think we can check `len(filters) == 1` and make error message like "single filter expected".
This would cover both chain and empty string case (not sure if ParseFilters fails on empty string or not). 

I also think we should not mention json/yaml in the error message as it is ambiguous and also routegroup could be created programmatically.

Reply:
for empty string we already validate with https://github.com/zalando/skipper/blob/master/dataclients/kubernetes/definitions/routegroupvalidator.go#L134-L140

Reply:
Since we check empty first `> 1` makes more sense because if the filter was invalid, `ParseFilters` will return empty filter which will then push 2 errors (parsing & single)

Reply:
Then maybe we should move empty check here to have filter validation in one place.

Note that value might also contain only whitespaces `"  "` which would produce zero filters but will not be empty - currently we do not catch this case. Here parse would correctly handle empty value like that